### DISCUSSION TITLE
Any YTDLnis alternative for Windows?

### DISCUSSION THREAD

**Discussion by ImMALWARE** [deniscerri/ytdlnis#315] [Q&A]
Hello guys, any YTDLnis alternative for Windows?

**Comment by deniscerri** [comment]
Install windows subsystem for Android and get the x86 apk

**Reply by ImMALWARE** [reply]
It is not what i want.

**Reply by deniscerri** [reply]
https://github.com/kannagi0303/yt-dlp-gui

**Reply by ImMALWARE** [reply]
Well, it can... Download videos and audios separately, and nothing else? Not like ytdlnis, that can everything. Well, will have to use this.
